src/index.css

```diff
@@ -1,27 +1,247 @@
+/* stylelint-disable value-list-comma-newline-after */
+/* stylelint-disable declaration-colon-newline-after */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+form button {
+  background-color: #4395fc;
+  border: none;
+  padding: 10px 20px;
+  color: white;
+  cursor: pointer;
+  width: 30%;
+  border-radius: 5px;
+  box-shadow: 5px 5px 5px rgb(0, 0, 0, 0.3);
+  font-weight: 600;
+}
+
+input {
+  border: solid 1px #e8e8e8;
+  padding: 10px;
+  margin-right: 20px;
+  background: white;
+}
+
+.header {
+  top: 30px;
+  align-items: center;
+  flex-direction: row;
+  display: flex;
+  border-bottom: 1px solid #e8e8e8;
+  padding: 3rem 6rem;
+  background-color: #fff;
+  gap: 50px;
+}
+
+.header h1 {
+  color: #0290ff;
+  font-family: Montserrat, sans-serif;
+  font-weight: 900;
+  font-stretch: normal;
+  font-size: 2.875rem;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
```

Comment:
- [ ] The `font-size` and the `font-weight` used for the `h1` element with the text `Bookstore CMS` is too much. I would suggest you reduce them so that your design looks like that of the Zeplin design. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208258780-f3fe5203-b2c5-4831-ade2-11913fed106f.png)

### Zeplin Design

![image](https://user-images.githubusercontent.com/77106962/208258649-5a945c26-0587-411b-b8e9-dc7f8f9263fd.png)